### PR TITLE
Bootstraping to LXD 2.0.x

### PR DESCRIPTION
**Pull request by SimonRichardson**
## Description of change

When using lxd 2.0.x that doesn't support api network extensions, 
the latest juju will fail because it can't find the configuration file 
inside the container. 

In current develop for Juju LXD 2.0.x will not bootstrap, the following
code will now bootstrap to LXD 2.0.x.

## QA steps

Install lxd 2.0 either via snap or apt-get, making sure that
you remove your current lxd 3.0 and your current lxd 2.0
setup. (this only presents itself on a clean install).

Installing lxd 2.0 for xenial:

```sh
sudo apt-get install lxd=2.0.11-0ubuntu1~16.04.4
```

```sh
juju bootstrap localhost test
```

## Bug reference

https://bugs.launchpad.net/juju/+bug/1788805

## Notes

This PR supersedes https://github.com/juju/juju/pull/8964

### CODE REVIEW

**Comment by SimonRichardson** [comment]
https://trello.com/c/khEJ5wsL/512-bug-1788805-lxd-20x-fail-to-bootstrap-with-latest-build

File: container/lxd/network.go [hunk 1]
```diff
@@ -239,7 +238,7 @@ func (s *Server) verifyNICsWithConfigFile(nics map[string]device, reader func(st
 			continue
 		}
 
-		logger.Infof("found usable network device %q with parent %q", name, netName)
+		logger.Debugf("found usable network device %q with parent %q", name, netName)
```
Comment (manadart) [inline container/lxd/network.go]: This output doesn't look to have changed in frequency, so this should be Tracef like on line 215 (I missed it before). In any case, they should both be the same.

**Comment by SimonRichardson** [comment]
!!build!!

**Comment by SimonRichardson** [comment]
!!build!!

**Comment by SimonRichardson** [comment]
!!build!!

**Comment by SimonRichardson** [comment]
!!build!!

Failed : https://bugs.launchpad.net/juju/+bug/1788401

**Comment by SimonRichardson** [comment]
!!build!!

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**